src/components/BalanceModule.js

```diff
@@ -1,33 +1,25 @@
-import React, { useCallback } from 'react'
-import { useHistory } from 'react-router-dom'
+import React from 'react'
```

Comment:
Remove the link to the deposit manager from the balance as it's not related. Especially when the user has a BYOT Garden.